nomad/fsm.go

```diff
@@ -1583,11 +1563,6 @@ func (n *nomadFSM) restoreImpl(old io.ReadCloser, filter *FSMFilter) error {
 		// Decode
 		snapType := SnapshotType(msgType[0])
 		switch snapType {
-		case TimeTableSnapshot:
```

Comment:
We actually need to keep this `case` as the `default` case crashes. We don't need to deserialize it, we just need to handle the case in order to prevent an `Unrecognized snapshot error`.

Reply:
oh that's a good catch

Reply:
We probably want to keep the const for a while as well to ensure we don't reuse its value and cause all kinds of fun problems.